EventListener/ViewResponseListener.php

```diff
@@ -96,7 +96,13 @@ public function onKernelView($event)
                 $context->disableMaxDepth();
             }
 
-            list($controller, $action) = $configuration->getOwner();
+            $owner = $configuration->getOwner();
+
+            if ([] === $owner) {
+                $owner = null;
+            }
+
+            [$controller, $action] = $owner;
```

Comment:
Maybe more understandable
```suggestion
            if ([] === $owner || null === $owner) {
                $controller = $action = null;
            } else {
                [$controller, $action] = $owner;
            }
```